### PR TITLE
Updated app example tutorial

### PR DESCRIPTION
**Pull request by michael-w-williams**
Removed triggers. Updated wording passes and such. Should be ready to go now.

### CODE REVIEW

File: Apps/README.md [hunk 1]
```diff
@@ -54,43 +67,60 @@ This will show the structure we've created which looks like this:
...
+```
 
 If you have previously set the `FN_REGISTRY` registry environment variable
 your Docker image names will be prefixed by it. Otherwise your output will look
 like:
 
 ```
-/       myapp2-root:0.0.2 localhost:8080/r/myapp2
-/hello  hello:0.0.2       localhost:8080/r/myapp2/hello
+fndemouser/hello        0.0.2       64049f5cfc52        10 minutes ago      15.6MB
```
Comment (rikgibson) [inline Apps/README.md]: This table doesn't appear to format quite correctly - the top line is out of kilter.

Comment (rikgibson) [reply Apps/README.md]: Otherwise looks pretty reasonable.

Comment (shaunsmith) [reply Apps/README.md]: I added a fix for the formatting.

**Comment by shaunsmith** [comment]
LGTM so I'm going to approve as I want to use this update in a tutorial today.

**Comment by michael-w-williams** [comment]
Thanks Rik and Shaun.

**Pull request merged**